src/reducers/sessionReducer.js

```diff
@@ -0,0 +1,27 @@
+import createReducer from 'reducers/createReducer';
+import { actionTypes } from 'actions/userActions';
+
+const initialState = {
+  user: null,
+  info: null,
+};
+
+const handleLoginSuccess = (state, { user }) => {
+  state.user = user;
+};
+
+const handleLogoutSuccess = state => {
+  state.user = null;
```

Comment:
I think here you can just do `return initialState;`

Reply:
I think I can't since this is an immer callback and we need to mutate the state without returning it.

Reply:
I am using immer and have this on logout success:
![image](https://user-images.githubusercontent.com/9297073/67225509-c647fb80-f409-11e9-9691-fcd156b1ed60.png)

Reply:
neat, will change it then